docs/_static/link_gen/link.js

```diff
@@ -58,22 +84,52 @@ var apps = {
     }
 }
 
+function changeTab(div) {
+    var hub = document.getElementById("hub");
+    var env_repo = document.getElementById("repo");
+    var content_repo = document.getElementById("content-repo");
+    var content_branch = document.getElementById("content-branch");
+    var id = div.id;
+
+    if (id.includes("mybinder")) {
+        hub.placeholder = "https://mybinder.org";
+        env_repo.labels[0].innerHTML = "Git Environment Repository URL";
+        content_repo.disabled = false;
+        content_branch.disabled = false;
+    } else {
+        hub.placeholder = "https://hub.example.com";
+        hub.disabled = false;
+        env_repo.labels[0].innerHTML = "Git Repository URL";
+        content_repo.disabled = true;
```

Comment:
What do you think about hiding rather than just disabling these in the other tabs? They only make sense for BinderHubs, and nothing else.